Target 'help' doesn't work (without venv)

The Makefile's `help` target fails if run without running one of the targets that create Python's virtual env. first:
```
/bin/sh: 1: sphinx-build: not found
make: *** [Makefile:14: help] Error 127
```
(There's no `SPHINXBUILD` to be had without it first being generated for the venv.)

Also, the command will fail even with venv created because the `sphinx-build` script is not in the PATH. So, the `activate` script needs to be sourced first like for the other targets that call `$(SPHINXBUILD)`.

--- a/Makefile
+++ b/Makefile
@@ -13,8 +13,8 @@ VENV          = $(VENVDIR)/bin/activate
 
 
 # Put it first so that "make" without argument is like "make help".
-help:
-	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+help: $(VENVDIR)
+	@. $(VENV); $(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
 .PHONY: help
 

